atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolingContainer.java

```diff
@@ -109,7 +109,6 @@ protected int getPoolSize() {
                 + "to ensure the TTransportException type is propagated correctly.");
     }
 
-    @SuppressWarnings("unchecked")
```

Comment:
looks like this warning was never there? the suppress seems to come from legacy code :/